### PR TITLE
Medical paperwork templates

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
- Added 4 paper templates for medical doctors with forms for: patient record, prescription, health insurance and medical invoice.
- Added folder and gauze to Nanomed Plus and wall-mounted Nanomed.

## Why / Balance
QoL

## How to test
1. Check `Fun` category in loadouts.
2. Spawn `medical doctor folder`, examine papers inside.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl: erhardsteinhauer
- add: Added a folder with paperwork templates for medical doctors to loadouts and to Nanomed Plus/Wall-mounted Nanomed.
- tweak: Added gauze to Nanomed Plus/Wall-mounted Nanomed catalogues.

### CODE REVIEW

**Comment by dvir001** [comment]
Loadout option?

**Comment by ErhardSteinhauer** [comment]
Would be nice, yeah. Sounds like a prime candidate for !!FUN!! category.

**Review by whatston3 — APPROVED** [review]
Looks alright, thanks for looking at the form changes.

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml [hunk 1]
```diff
@@ -16,119 +16,162 @@
...
-        [head=2]Section A: General Information[/head]
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Health Insurance Contract[/head]
+
+      [head=3]Policy Holder Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Occupation:[/bold]
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml:123]:
```suggestion
          [bold]Occupation:[/bold] 
          [bold]Vessel IFF:[/bold] 
```

Forgot about any associated ship you're interested in here, potentially useful in triage.

**Comment by dvir001** [comment]
We can probably add a new category called "docs" for RP and have people select from max one, so it wont effect the "Fun" options

**Comment by ErhardSteinhauer** [comment]
> We can probably add a new category called "Docs" or something like this, for RP and have people select from max one, so it wont effect the "Fun" options

Maybe. But I believe paperwork is firmly on HRP territory. And I'm not sure if it's worth creating a full category for 3 items - SR folder, medical folder and nfsd folder (don't think there are any other player factions that require standardized paperwork, might be wrong here).

**Comment by ErhardSteinhauer** [comment]
Probably can put folders with paperwork in `trinkets` category and increase the upper limit there.

**Comment by ErhardSteinhauer** [comment]
Or! Or we can create a `Bureaucracy` category for all the stamps players want and drop folders with paperwork there.

**Comment by dvir001** [comment]
> Or! Or we can create a `Bureaucracy` category for all the stamps players want and drop folders with paperwork there.

Bureaucracy sound fine,

STC
SR
Medical
Nfsd
Etc

Let people just add stuff as it goes I think its fine to have this as a category.

For now you can yeet the medical one in it.

**Comment by whatston3** [comment]
> > Or! Or we can create a `Bureaucracy` category for all the stamps players want and drop folders with paperwork there.
> 
> Bureaucracy sound fine,
> 
> STC SR Medical Nfsd Etc
> 
> Let people just add stuff as it goes I think its fine to have this as a category.
> 
> For now you can yeet the medical one in it.

I've put up a set of changes for this loadout stuff here:

https://github.com/ErhardSteinhauer/frontier-station-14/compare/2024-07-Medcal-Forms...whatston3:frontier-station-14:2024-07-Medcal-Forms-suggestions?expand=1

Should probably be reviewed as a separate feature, but it includes squiggly signature sprites, stamps for doctors and engineers (for invoices/signoff RP), a bureaucracy loadout setup, and some cleanup better denoting NF stamps.

I think this branch as it is is good to go.

image

**Comment by ErhardSteinhauer** [comment]
> > > Or! Or we can create a `Bureaucracy` category for all the stamps players want and drop folders with paperwork there.
> > 
> > 
> > Bureaucracy sound fine,
> > STC SR Medical Nfsd Etc
> > Let people just add stuff as it goes I think its fine to have this as a category.
> > For now you can yeet the medical one in it.
> 
> I've put up a set of changes for this loadout stuff here:
> 
> https://github.com/ErhardSteinhauer/frontier-station-14/compare/2024-07-Medcal-Forms...whatston3:frontier-station-14:2024-07-Medcal-Forms-suggestions?expand=1
> 
> Should probably be reviewed as a separate feature, but it includes squiggly signature sprites, stamps for doctors and engineers (for invoices/signoff RP), a bureaucracy loadout setup, and some cleanup better denoting NF stamps.
> 
> I think this branch as it is is good to go.
> 
> image

'Ery noice.

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+      [head=3]Responder Information[/head]
+            [bold]Name:[/bold] 
+            [bold]Vessel IFF:[/bold] 
+            [bold]Shift Time:[/bold] 
+
+      [head=3]Invoice[/head]
+      - $XX XXX - Treatment
+      - $XX XXX - Revival
+      - $XX XXX - Cloning
+      - $XX XXX - Towing
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml:168]:
```suggestion
      $XX XXX - Treatment
      $XX XXX - Revival
      $XX XXX - Cloning
      $XX XXX - Towing
```

The idea here is that these are additive (summing up your cost), but the coverage should be negative (covering all/part of your costs).  Is there a better way to do this?

```
       $XX XXX - Treatment
       $XX XXX - Revival
       $XX XXX - Cloning
       $XX XXX - Towing
      ($XX XXX) - Treatment Plan Coverage
      ($XX XXX) - Revival Plan Coverage
      ($XX XXX) - Cloning Plan Coverage
      ($XX XXX) - Towing Plan Coverage
```

File: Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+          [bold]Medical Coverage:[/bold]
+          \[  ] Treatment: Coverage up to $5 000. ($2 000 premium)
+          \[  ] Revival: Coverage up to $10 000. ($2 000 premium)
+          \[  ] Cloning Fee: Coverage up to $25 000. ($5 000 premium)
+
+          [bold]Additional Coverage:[/bold]
+          \[  ] Towing: Coverage up to $10 000. Small shuttles only. ($2 000 premium)
+
+      [head=3]Grand Total[/head]
+          [bold]$XXX XXX[/bold]
```
Comment (whatston3) [inline Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml:136]: Might be annoying, but I think saying this is the total of the premium vs. the coverage (which could also be listed, but I don't think it's critical)
```suggestion
      [head=3]Total Premium[/head]
          [bold]$XX XXX[/bold]
```

**Review by whatston3 — CHANGES_REQUESTED** [review]
Another review, another few requests.  Sorry.

**Review by whatston3 — APPROVED** [review]
It is good.

**Review by whatston3 — APPROVED** [review]
Think that should be it.  Recommending this to be merged.

**Comment by whatston3** [comment]
Erhard's signed off on this, gonna merge it.

**Pull request merged**